src/JSInterop/Microsoft.JSInterop/src/IJSInProcessObjectReference.cs

```diff
@@ -1,12 +1,14 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
+
 namespace Microsoft.JSInterop
 {
     /// <summary>
     /// Represents a reference to a JavaScript object whose functions can be invoked synchronously.
     /// </summary>
-    public interface IJSInProcessObjectReference : IJSObjectReference
+    public interface IJSInProcessObjectReference : IJSObjectReference, IDisposable
```

Comment:
I thought it made sense that `IJSInProcessObjectReference`s should be able to be disposed synchronously.

Reply:
Very good point